third_party/terraform/tests/resource_google_service_account_test.go

```diff
@@ -52,7 +52,7 @@ func TestAccServiceAccount_basic(t *testing.T) {
 			},
 			// The second step updates the service account
 			{
-				Config: testAccServiceAccountBasic(accountId, displayName2, desc2),
+				Config: testAccServiceAccountBasic(accountId, displayName2, description2),
```

Comment:
Why this change? It's really helpful for us to know that we can set things to empty, since that case often accidentally doesn't work.